Make special note lookup keybinds more obvious (#286)
dendronhq/dendron-site: vault/dendron.topic.special-notes.md

A journal note is a self contained note that is meant to track something over time. Examples of journals include recording **workout sessions**, making **meeting notes**, and keeping a **mood journal**.

> To create a journal note, trigger a lookup and then click on the calendar icon, or use the keybind "ctrl+shift+j"/"cmd+shift+j" (depending on operating system).

Reviewer: ```suggestion
To create a journal note, trigger a lookup and then click on the calendar icon, or use the `Ctrl+Shift+J`/`Cmd+Shift+J` shortcut (depending on operating system).
```

## Scratch Note

> A scratch note is a self contained note that is meant to be used as scratchpad. Use it for thoughts or when you want to expand on a bullet point. Scratch notes are created in the `scratch` domain and have the following format: `scratch.{y.MM.dd.HHmmss}`. To create a scratch note, trigger a lookup and then click the "create scratch note" icon, or use the keybind "ctrl+shift+s"/"cmd+shift+s" (depending on operating system).

Reviewer: ```suggestion
A scratch note is a self contained note that is meant to be used as scratchpad. Use it for thoughts or when you want to expand on a bullet point. Scratch notes are created in the `scratch` domain and have the following format: `scratch.{y.MM.dd.HHmmss}`. To create a scratch note, trigger a lookup and then click the "create scratch note" icon, or use the keybind `Ctrl+Shift+S`/`Cmd+Shift+S` (depending on operating system).
```

<a href="https://www.loom.com/share/2fd3042119124df8bb4592d8ffe6d708"> 
<img style="" src="https://cdn.loom.com/sessions/thumbnails/2fd3042119124df8bb4592d8ffe6d708-with-play.gif"> </a>

## Configuration

> To change the keybindings used to create special notes, you can [define custom shortcuts](https://wiki.dendron.so/notes/66d734d2-07bd-4ac6-be0b-a38b55bc1099/) in `keybindings.json` that will [modify the behaviour of the lookup command](https://wiki.dendron.so/notes/ad270a7d-2aed-4273-8319-eb6536e38b29/).

Reviewer: ```suggestion
To change the keybindings used to create special notes, you can [[define custom shortcuts|dendron.topic.lookup.keybindings]] in `keybindings.json` that will [[modify the behavior of the `Lookup` command|dendron.topic.lookup.modifiers]].
```
